libs/src/services/creatorNode/CreatorNodeSelection.js

```diff
@@ -120,7 +120,7 @@ class CreatorNodeSelection extends ServiceSelection {
     const secondaries = this.getSecondaries(services, primary)
     this.decisionTree.push({
       stage: DECISION_TREE_STATE.SELECT_PRIMARY_AND_SECONDARIES,
-      val: { primary, secondaries: secondaries.toString(), services: servicesMap }
+      val: { primary, secondaries: secondaries.toString(), services: JSON.stringify(Object.keys(servicesMap)) }
```

Comment:
why JSON.stringify this?

Reply:
ah, I must have left it in because originally I was printing out the map, not the keys. should be able to safely remove